domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3354,9 +3482,28 @@ public void getConnection(Handler<AsyncResult<PgConnection>> replyHandler) {
    * @see #withTrans(Function)
    * @see #withTransaction(Function)
    */
+
   void getSQLConnection(Handler<AsyncResult<SQLConnection>> handler) {
     getSQLConnection(0, handler);
   }
+  /**
+   * Don't forget to close the connection!
+   *
+   * <p>Use closeAndHandleResult as replyHandler, for example:
+   *
+   * <pre>getSQLReadConnection(conn -> execute(conn, sql, params, closeAndHandleResult(conn, replyHandler)))</pre>
+   *
+   * <p>Or avoid this method and use the preferred {@link #withConn(Function)}.
+   *
+   * @see #withReadConn(Function)
+   * @see #withReadConnection(Function)
+   * @see #withTrans(Function)
+   * @see #withTransaction(Function)
```

Comment:
`withTrans` should be `withReadTrans`
`withTransaction` shoudl be `withReadTransaction`